deploy/deployArrowTokenL2.js

```diff
@@ -0,0 +1,41 @@
+const L2StandardTokenFactoryArtifact = require(`@eth-optimism/contracts/artifacts/contracts/L2/messaging/L2StandardTokenFactory.sol/L2StandardTokenFactory.json`);
+
+const func = async (hre) => {
+  const L1TokenAddress = process.env.L1_TOKEN_ADDRESS;
+  const L2TokenName = process.env.L2_TOKEN_NAME;
+  const L2TokenSymbol = process.env.L2_TOKEN_SYMBOL;
+
+  // Instantiate the signer
+  const provider = new ethers.providers.JsonRpcProvider(hre.network.config.url);
+  const signer = new ethers.Wallet(process.env.DEPLOYMENT_PRIVATE_KEY, provider);
+
+  console.log(
+    "Creating instance of L2StandardERC20 on",
+    hre.network.name,
+    "network"
+  );
+  // Instantiate the Standard token factory
+  const l2StandardTokenFactory = new ethers.Contract(
+    "0x4200000000000000000000000000000000000012",
+    L2StandardTokenFactoryArtifact.abi,
+    signer
+  );
```

Comment:
This can only be run on Optimism, so a check that we are running on the right chain would be useful here.